client/src/app/metadata-schema-form/metadata-field-types/text-area/text-area.component.html

```diff
@@ -12,8 +14,5 @@
     <p class="vf-form__helper vf-form__helper--error">{{checkForErrors(control)}}</p>
 
   </div>
-  <div>
-    <p *ngIf="helperText" class="vf-form__helper">{{helperText}}</p>
 
-  </div>
```

Comment:
the helper text is being removed for text area fields

Reply:
I think you forgot to update this to use form item component

Reply:
This component isn't actually being used anywhere, but I've reverted it to it's previous version.